assets/dev/js/frontend/handlers/base-tabs.js

```diff
@@ -50,6 +67,53 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 		this.setSettings( originalToggleMethods );
 	}
 
+	handleKeyboardNavigation( event ) {
+		const key = event.keyCode,
+			tab = event.currentTarget,
+			keys = this.getSettings( 'keys' ),
+			$tabList = jQuery( tab.closest( this.getSettings( 'selectors' ).tablist ) ),
+			$tabs = $tabList.find( this.getSettings( 'selectors' ).tabTitle ),
```

Comment:
Check if you can store these elements inside `getDefaultElements()`

Reply:
They depend on the event. (const `tab`)
The only way I could store them on the object level, is with a callback, not literally.